Improve Installation Doc Readability based for Step 3 relating to GPU setup (#1037)
ivadomed/ivadomed: docs/source/installation.rst

    .. tabs::

>         .. tab:: PyPI Installation

Reviewer: What do you think about linking these two sets of tabs? 

Sphinx-tabs has a [Group Tabs](https://sphinx-tabs.readthedocs.io/en/latest/#group-tabs) feature that lets you change all the linked tabs at once. (We could also do that for the Mac/Linux and Windows tabs, too.)

Author: SICK yo. That is such a cool trick! OMG. So cool. I will definitely use it and adopt it for others. TIL!

            .. tabs::

                .. tab:: Nvidia GPU Support

>                     ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.

Reviewer: It might be worthwhile to instruct people to run `nvidia-smi` to check their CUDA and driver versions:

![Screenshot from 2022-01-10 13-16-28](https://user-images.githubusercontent.com/16181459/148817710-565e0b9a-d4bf-4941-91b6-ca926133c1d3.png)

Author: Great point! Added two more sentences to provide brief resources to help people out with that.

>                     If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed try run the following command relevant to your situation:

Reviewer: Nitpick, but the wording "try run the following command relevant to your situation" feels a bit unnecessary?

I think we could cut it down to just:

```suggestion
                    If you have a compatible NVIDIA GPU that supports CUDA11, and you have a recent enough driver installed, then run the following command:
```

Author: Fixed. I am a wordy person. :D

                    .. code::

                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html

                .. tab:: CPU Support

                    If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:

                    .. tabs::

                        .. tab:: Windows/Linux

                            .. code::

                               pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html

                        .. tab:: Mac

                            .. code::

                               pip install torch==1.8.0 torchvision==0.9.0 --find-links https://download.pytorch.org/whl/torch_stable.html

        .. tab:: Repo Installation (Advanced or Developer)

            Run this only if you have already downloaded/cloned the repo with access to the ``requirement_gpu.txt`` file, then run the following command while at the repository root level:
            .. code::

>                pip install -r requirements_gpu.txt

Reviewer: This `.. code::` bit isn't rendering correctly for me:

![Screenshot from 2022-01-10 13-07-47](https://user-images.githubusercontent.com/16181459/148816626-5ce66f33-9a48-4f48-9d79-537368e8503c.png)

Author: Fixed. Thanks!

    .. tabs::

        .. group-tab:: PyPI Installation

            .. tabs::

                .. tab:: Nvidia GPU Support

                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`__, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.

>                     You can use ``nvidia-smi`` in both Linux and Windows to check for driver CUDA Version listed at the top right of the output console. On windows, follow `this example guide here until Step 4 <https://wiki.tuflow.com/index.php?title=DOS_GPU_Usage>`__. On Linux, simply type in ``nvidia-smi`` in any console to see the output.

Reviewer: I think this Windows tutorial link might be out of date? According to [this StackOverflow comment](https://stackoverflow.com/a/57100016), the location of `nvidia-smi` seems to vary depending on how recent the version is. 

So, we might want to word it like this:

```suggestion
                    You can use ``nvidia-smi`` in both Linux and Windows to check for driver CUDA Version listed at the top right of the output console. On Linux, simply type in ``nvidia-smi`` in any console to see the output. On windows, you will need to locate the `nvidia-smi.exe` tool by following the instructions on `this page <https://stackoverflow.com/a/57100016>`__. 
```
